Source/KSCrash/Source/KSCrash/Recording/BSG_KSSystemInfo.h

```diff
@@ -57,7 +57,9 @@
 #define BSG_KSSystemField_BuildType "build_type"
 
 #import <Foundation/Foundation.h>
```

Comment:
Should we remove the `Foundation/Foundation.h` include?

Reply:
This is an import which means Obj-C will guard against the header being included more than once. It's also more explicit that `BSG_KSSystemInfo` requires this header and guards against the internals of `BugsnagPlatformConditional` removing `Foundation.h` at some later date and then having to add `Foundation.h` back in again.